src/reporter.ts

```diff
@@ -64,8 +65,13 @@ export const defaultReporter = <T = any>(
   if (onError) {
     onError(errors)
```

Comment:
Doesn't providing an onError (above) instead solve this problem already? It can be used to prevent the process.exit(1) call below. TBH I don't really see why we need another option to the reporter

Reply:
It's not, the defaultReporter isn't really advertised in the documentation. Customizing it is more of an advanced use case. But I do think you can use it instead of adding errorMode for what you're trying to achieve